packages/graphql/tests/integration/directives/relationship/duplicate-relationship/duplicate-relationship-read-interface.int.test.ts

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { UniqueType } from "../../../../utils/graphql-types";
+import { TestHelper } from "../../../../utils/tests-helper";
+
+describe("Interface: Multiple relationships results difference between Connection API and Simple API", () => {
+    const testHelper: TestHelper = new TestHelper();
+
+    const Production: UniqueType = testHelper.createUniqueType("Production");
+    const Movie: UniqueType = testHelper.createUniqueType("Movie");
+    const Series: UniqueType = testHelper.createUniqueType("Series");
+    const Actor: UniqueType = testHelper.createUniqueType("Actor");
+
+    beforeAll(async () => {
+        const typeDefs = /* GraphQL */ `
+            interface ${Production}  {
+                title: String!
+            }
+
+            type ${Movie} implements ${Production} @node {
+                title: String!
+                actors: [${Actor}!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+            }
+
+            type ${Series} implements ${Production} @node {
+                title: String!
+                actors: [${Actor}!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+            }
+
+            type ${Actor} @node {
+                name: String!
+                productions: [${Production}!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: OUT)
+            }
+
+            type ActedIn @relationshipProperties {
+                role: String!
+            }
+        `;
+        await testHelper.initNeo4jGraphQL({ typeDefs });
+
+        // Deliberately create duplicate relationships
```

Comment:
```suggestion
        //  Create duplicate relationships
```
Nitpick: `Deliberately` to me seems imply a case not expected